XiEditor/Search.swift

```diff
@@ -228,9 +228,8 @@ extension EditViewController {
             Swift.print("replaceAndFind not implemented")
 
         case .setSearchString:
-            openFind()
-            self.find(nil, caseSensitive: !findViewController.ignoreCase, regex: findViewController.regex)
-
+            document.sendRpcAsync("selection_for_find", params: ["case_sensitive": false])
```

Comment:
Not sure whether it should be case sensitive (Sublime behaviour) or case insensitive (macOS). For now I set it to be case insensitive.

Reply:
I think Sublime uses whatever the previous setting in the find panel is/was? However I like adding this option, since it gives us more flexibility down the road.